### PR TITLE
remove ddoc whitespace

### PR DESCRIPTION
**Pull request by wilzbach**
follow-up to #4354 - a couple of other files in which ddoc insert unnecessary whitespace that looks ugly.

### CODE REVIEW

File: std/csv.d [hunk 1]
```diff
@@ -52,10 +51,8 @@
  }
  * }
  * -------
- *
```
Comment (wilzbach) [inline std/csv.d]: I started to fix that, but realized it's not a good idea and should better be fixed directly in Ddoc. After a unittest there should be one empty line allowed without inserting any new line.

File: std/experimental/logger/package.d [hunk 2]
```diff
@@ -4,9 +4,8 @@ Implements logging facilities.
 Copyright: Copyright Robert "burner" Schadek 2013 --
 License: <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
 Authors: $(WEB http://www.svs.informatik.uni-oldenburg.de/60865.html, Robert burner Schadek)
-
-$(H3 Basic Logging)
-
+$(H3 Basic Logging
+)
```
Comment (wilzbach) [inline std/experimental/logger/package.d]: I know this is ugly and better ideas to fix ddoc are welcome.
Please have a look at the rendered output:
https://dlang.org/phobos/std_experimental_logger.html

File: std/base64.d [hunk 3]
```diff
@@ -24,7 +24,6 @@
  * ubyte[] decoded = Base64.decode("FPucA9l+");
  * assert(decoded == [0x14, 0xfb, 0x9c, 0x03, 0xd9, 0x7e]);
  * -----
- *
```
Comment (aG0aep6G) [inline std/base64.d]: This doesn't seem to have any effect.

Comment (aG0aep6G) [inline std/base64.d]: > This doesn't seem to have any effect.

Scratch that. It doesn't have an effect on the DDOX page, but it has an effect on the Ddoc page.

**Comment by aG0aep6G** [comment]
This really should be fixed in dmd, not here.

**Comment by wilzbach** [comment]
closing in favor of (hopefully) fixing ddocs
https://issues.dlang.org/show_bug.cgi?id=16110

**Pull request closed without merging**